ci_framework/roles/os_net_setup/tasks/main.yml

```diff
@@ -0,0 +1,37 @@
+# assiming having a login session with the right namespace already
+- name: get openstack admin credentials from k8s
+  block:
+  - name: Get clouds.yaml configuration
+    ansible.builtin.command:
+      cmd: oc get configmaps openstack-config -o jsonpath={.data}
+    register: clouds_yaml
+    environment:
+      PATH: "{{ path }}"
+
+  - name: Get the OSP admin password
+    ansible.builtin.command:
+      cmd: oc get secrets openstack-config-secret -o jsonpath={.data}
+    register: osp_secret
+    environment:
+      PATH: "{{ path }}"
+  vars:
```

Comment:
please move this right after the `block`- that will be easier to ready and follow.